packages/client/src/devices/InputMediaDeviceManager.ts

```diff
@@ -63,16 +63,23 @@ export abstract class InputMediaDeviceManager<
    * Starts stream.
    */
   async enable() {
-    if (this.state.status === 'enabled') return;
-    this.enablePromise = this.unmuteStream();
-    try {
-      await this.enablePromise;
-      this.state.setStatus('enabled');
-      this.enablePromise = undefined;
-    } catch (error) {
-      this.enablePromise = undefined;
-      throw error;
-    }
+    if (this.state.pendingStatus === 'enabled') return;
```

Comment:
1. If there already is a pending request to enable device - do nothing, the device will end up enabled anyway.